test/integration/targets/copy/tasks/main.yml

```diff
@@ -28,6 +25,9 @@
     that:
       - "file_result_check.mode == '0444'"
 
+- set_fact:
+    output_dir_expanded: '{{ file_result_check.path | dirname }}'
+
```

Comment:
I think this is okay given the way that we currently use it but it doesn't seem right to set a variable that will be used to check correctness of the module from the module itself so we probably shouldn't do it this way (it makes it so there are certain things we can't use this variable for which is a trap for the unwary).  Maybe 
```
command: 'echo {{ output_dir }}'
```
or something else like that would be a better way to set this.